pkg/chains/formats/intotoite6/intotoite6.go

```diff
@@ -286,5 +289,18 @@ func gitInfo(tr *v1beta1.TaskRun) (commit string, url string) {
 			url = r.Value
 		}
 	}
+
+	url = spdxGit(url, "")
```

Comment:
we don't have to validate url do we? It's pretty much expected to be a fully formed url?

Reply:
yah, i think it's expected!